mappings/net/minecraft/component/type/PotionContentsComponent.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/unmapped/C_xpqegwnr net/minecraft/component/type/PotionContentsComponent
+	FIELD f_ikryleaj NO_EFFECT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_iuaftjtj EMPTY_COLOR I
+	FIELD f_nxpdykpa FULL_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_odxdsbyn BASE_POTION_COLOR I
+	FIELD f_pfmdfdye CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_srqdzwer NO_POTION_COLOR I
+	FIELD f_ujtaryrq DEFAULT Lnet/minecraft/unmapped/C_xpqegwnr;
+	METHOD m_cwohitpj getArrowColor ()I
+	METHOD m_hbrdlxey getEffects ()Ljava/lang/Iterable;
+	METHOD m_hcngmcff createStack (Lnet/minecraft/unmapped/C_vorddnax;Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_sddaxwyk;
+	METHOD m_ibnobein buildTooltip (Ljava/util/function/Consumer;FF)V
+	METHOD m_ivpikusu withPotion (Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_xpqegwnr;
+	METHOD m_jtikrmsj (Ljava/util/List;Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_hdbqsqsm;)V
+		ARG 1 effect
+		ARG 2 modifier
+	METHOD m_linsoxal getColor (Ljava/lang/Iterable;)I
+	METHOD m_lwwcxstm getColor (Lnet/minecraft/unmapped/C_cjzoxshv;)I
+		ARG 0 potion
+	METHOD m_mgymqmlt customEffects ()Ljava/util/List;
+	METHOD m_nghpasjm (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+		ARG 0 instance
+	METHOD m_njngtrku matches (Lnet/minecraft/unmapped/C_cjzoxshv;)Z
+		ARG 1 potion
+	METHOD m_nlaiylfj getColor ()I
+	METHOD m_oixxcdng getColorOptional (Ljava/lang/Iterable;)I
```

Comment:
```suggestion
	METHOD m_oixxcdng tryGetColor (Ljava/lang/Iterable;)I
```
?

Reply:
I prefer the `optional` suffix over the `try` prefix for methods like this

Reply:
Would you consider `getOptionalColor` (and `optionalFromNbt`)?
I think having the adjective follow the noun makes it more awkward to read.

Reply:
discussed on discord